conda-envs/environment-jax.yml

```diff
@@ -11,9 +11,9 @@ dependencies:
 - cloudpickle
 - h5py>=2.7
 # Jaxlib version must not be greater than jax version!
-- blackjax==1.2.0  # Blackjax>=1.2.1 is incompatible with latest available version of jaxlib in conda-forge
-- jaxlib==0.4.23  # Latest available version in conda-forge, update when new version is available
-- jax==0.4.23
+- blackjax
+- jaxlib
+- jax
```

Comment:
We still need to pin to the same number no?

Reply:
My experience in blackjax is that the version is automatically resolved.

Reply:
Can we lower pin the versions to the latest?